corehq/apps/cleanup/management/commands/fix_xforms_with_undefined_xmlns.py

```diff
@@ -26,6 +28,9 @@
 SET_XMLNS = "set-xmlns-on-xfrom"
 ERROR_SAVING = "error-saving-xform"
 MULTI_MATCH = "multiple-possible-forms"
+NONE_MARKED_FIXED = "no-forms-in-build-marked-fixed"
```

Comment:
Ah thanks, this was a less specific message left over from an earlier version of the command that I refactored out. I'll remove it.